### PR TITLE
Execution profiles doc

### PR DESCRIPTION
**Pull request by jorgebay**

### CODE REVIEW

File: doc/features/execution-profiles/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Execution Profiles (experimental)
+
+Execution profiles provide a mechanism to group together a set of configuration options and reuse them across different
```
Comment (tolbertam) [inline doc/features/execution-profiles/README.md]: Nice overview, sets the expectations well.

File: doc/features/execution-profiles/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Using Execution Profiles
+
+### Initializing cluster with profiles
+
+Execution profiles should be created when creating the `Client` instance with a name that identifies it.
+
+```javascript
+const metricsProfile = new ExecutionProfile('metrics', {
+  consistency: consistency.localQuorum,
```
Comment (tolbertam) [inline doc/features/execution-profiles/README.md]: For demonstration it would be good to use all profile options here.

Comment (jorgebay) [inline doc/features/execution-profiles/README.md]: good call, will add the rest.

**Comment by tolbertam** [comment]
👍

**Pull request merged**